src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -103,7 +102,7 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
     this.setState({
       fields: [
         { label: 'All', value: ALL_VARIABLE_VALUE },
-        ...getUniqueFilters(logsScene, logsScene.state.detectedFields || []).map((f) => ({
+        ...(logsScene.state.detectedFields || []).map((f) => ({
```

Comment:
nit: Nullish coalescing operator `??` instead of boolean or `||`

Reply:
Fixed in https://github.com/grafana/explore-logs/pull/449/commits/d81bd33b4da18389fd4742f74f4f7d53e156a43f